electron-app/src/components/inputs/AssetSelect.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <v-autocomplete
+    :value="value"
+    :disabled="disabled"
+    :items="supportedAssets"
+    class="asset-select"
+    single-line
+    label="Asset"
+    :rules="rules"
+    item-text="symbol"
+    :menu-props="{ closeOnClick: true, closeOnContentClick: true }"
+    item-value="symbol"
+    @input="input"
+  >
+    <template #selection="{ item }">
+      <span>{{ item.symbol }}</span>
+    </template>
+    <template #item="{ item }">
+      <v-list-item-avatar>
+        <crypto-icon :symbol="item.symbol"></crypto-icon>
```

Comment:
Okay so now here we may have a problem.

As I wrote in the docs the symbol may not be unique. BUT it is indeed what the assets is known as. Unique is only going to be the key of the mapping.

Check for example the 3 different assets with the symbol `ACC` at `all_assets.json`:

https://github.com/rotki/rotki/blob/14b895bab0fad654d7abbacd2d1708255297e26e/rotkehlchen/data/all_assets.json#L94-L114

For now to guarantee uniqueness we make sure the keys are following the `SYMBOL-N` if multiple assets use a symbol.

The problem I see with the approach here is that the UI would show 3 different assets with `ACC` and the user can not know which is which.

One idea here is. Can we -- only for assets whose key contains `-N`, meaning multiple assets with the same symbol exist -- also add the name?

EDIT: Looking at the actual result of typing `ACC` into the search it seems even worse. Only 1 ACC is returned and the others are ignored.

![2020-04-12-004332_1204x263_scrot](https://user-images.githubusercontent.com/1658405/79056417-bf7ea080-7c56-11ea-81a3-d44506e99d19.png)